seqr/utils/elasticsearch/es_utils_2_3_tests.py

```diff
@@ -16,10 +16,12 @@
 from seqr.utils.elasticsearch.es_search import EsSearch, _get_family_affected_status, _liftover_grch38_to_grch37, \
     _liftover_grch37_to_grch38
 from seqr.views.utils.test_utils import PARSED_VARIANTS, PARSED_SV_VARIANT, TRANSCRIPT_2
+from seqr.utils.redis_utils import safe_redis_get_json
```

Comment:
Codacy found an issue: [Unused safe_redis_get_json imported from seqr.utils.redis_utils](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5859170)

Reply:
this needs to be removed